fairmq/shmem/Region.cxx

```diff
@@ -49,7 +50,16 @@ Region::Region(Manager& manager, uint64_t id, uint64_t size, bool remote, Region
     if (path != "") {
         fName = string(path + fName);
 
-        fFile = fopen(fName.c_str(), fRemote ? "r+" : "w+");
+        if (!fRemote) {
+            // create a file
+            filebuf fbuf;
+            fbuf.open(fName, ios_base::in | ios_base::out | ios_base::trunc | ios_base::binary);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Return value of function fbuf.open() is not used.](https://app.codacy.com/manual/FairRootGroup/FairMQ/pullRequest?prid=5435982)